fix

TerableCoder opened the pull request.

TerableCoder committed:
old working version

TerableCoder committed:
new working version

TerableCoder committed:
fixed

TerableCoder committed:
Update README.md

TerableCoder committed:
ops

TerableCoder committed:
ufc or ucf? doesn't matter!

Works regardless of if you have your S_FATIGABILITY_POINT.3.def looking like 
int32 unk
int32 fatigability
int32 current

or
int32 unk
int32 current
int32 fatigability

TerableCoder committed:
added gif

TerableCoder commented:
Works regardless of packet labeling.

Sunpui commented:
am I seeing this correct? Are you trying to use cures while you're in a crafting animation?

Sunpui commented:
oh they changed that nvm

TerableCoder committed:
toggling the module off then on no longer unlocks

toggling the module off then on no longer unlocks
added delay option
many small changes
